<?xml version="1.0" ?>

<launch>    
	
    <param name="offset_file_path"  value="$(find thormang3_manager)/config/offset.yaml"/>	
    <param name="robot_file_path"   value="$(find thormang3_manager)/config/THORMANG3.robot"/>
    <param name="init_file_path"    value="$(find thormang3_manager)/config/dxl_init.yaml"/>   
    	
    <node name="thormang3_offset_tuner_server_node" pkg="thormang3_offset_tuner_server" type="thormang3_offset_tuner_server_node" output="screen">
    </node>
    
</launch>
